### PR TITLE
HW 11.Kubernetes_installation + HW 12.Kubernetes.First_deployment to md-sa2-20-22

### PR DESCRIPTION
**Pull request by artem-devitsky**

### CODE REVIEW

**Comment by artem-devitsky** [comment]
For HW 12 I reused workflow that count tabs in commits.
https://github.com/artem-devitsky/12.Kubernetes.First_deployment

File: Artsiom_Dziavitski/12.Kubernetes.First_deployment/manifests/self-hosted-runner/01-cert-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,5388 @@
+# Copyright 2021 The cert-manager Authors.
```
Comment (pluhin) [inline Artsiom_Dziavitski/12.Kubernetes.First_deployment/manifests/self-hosted-runner/01-cert-manager.yaml:1]: looks like to many for one runner, it is very good you find this solution, but to many resources, manifests for one runner

Comment (pluhin) [reply Artsiom_Dziavitski/12.Kubernetes.First_deployment/manifests/self-hosted-runner/01-cert-manager.yaml:1]: please do not deploy on our servers, too heavy for them

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**